index.html

```diff
@@ -7,70 +7,412 @@
+                    <h2 class="dark-blue">Hey there.I’m Farid  <br> <span class="green">I’m a software developer</span></h2>
                 </header>
-                <p>
+                <h4 class="blue center">
                     I can help you build a product , feature or website Look through some of my work and experience! If you
                     like
                     what you see and have a project you need coded, <br>don’t hestiate to contact me.
-                </p>
+                </h4>
             </section>
         </section>
-        <footer>      
+        <section class="links">      
                 <ul>
                     <li>
                         <a href="#">
-                            <i class="fa fa-twitter" aria-hidden="true"></i>
+                            <i class="fa fa-2x fa-twitter" aria-hidden="true"></i>
                         </a>
                     </li>
                     <li>
                         <a href="#">
 
-                            <i class="fa fa-linkedin" aria-hidden="true"></i>
+                            <i class="fa fa-2x fa-linkedin" aria-hidden="true"></i>
                         </a>
                     </li>
                     <li>
                         <a href="#">
 
-                            <i class="fa fa-medium" aria-hidden="true"></i>
+                            <i class="fa fa-2x fa-medium" aria-hidden="true"></i>
                         </a>
                     </li>
                     <li>
                         <a href="#">
-                            <i class="fa fa-github" aria-hidden="true"></i>
+                            <i class="fa fa-2x fa-github" aria-hidden="true"></i>
                             
                         </a>
                     </li>
                     <li>
                         <a href="#">
-                            <i class="fa fa-angellist" aria-hidden="true"></i>
+                            <i class="fa fa-2x fa-angellist" aria-hidden="true"></i>
 
                         </a>
                     </li>
                 </ul>
                     <a href="#">
                         <i><img src="./assets/Disabled.png" alt=""></i>
                     </a>             
-        </footer>
+                </section>
     </section>
+
+    <main>
+        <section class="projects">
+            <header class="centered-column">
+                <h2 class="dark-blue">
+                    My Recent <br> Works
+                </h2>
+                <hr class="bg-green card hr">
+            </header>
+            <section class="work">
+
+                <section class="bordered-card">
+
+                    <section class="stretched-column card">
+                        <figure>
+
+                            <img src="../assets/place_holders/plaholder.png" class="bg-light-green place-holder" alt="">
+
+                        </figure>
```

Comment:
I love the design and implementation of the template you chose 👍. However, I would like to recommend some changes so that your project can be better

- [ ] The image is broken and is not showing. Kindly see the screenshot below and fix it

![image](https://user-images.githubusercontent.com/77106962/143584418-0c153f35-6cf4-4ce7-b80c-0f386ade147f.png)

- [ ] It is considered good practice to put a text that describes the image in the alt attributes. This makes it easy for developers and users of your site who are physically impaired to understand what you are doing with the aid of a screen reader 🙏 .